app/src/main/java/app/revanced/manager/ui/screen/AppSelectorScreen.kt

```diff
@@ -59,6 +64,17 @@ fun AppSelectorScreen(
     onBackClick: () -> Unit,
     vm: AppSelectorViewModel = koinViewModel()
 ) {
+    val androidContext = LocalContext.current
+
+    if (vm.android11BugActive)
+        PermissionRequestDialog(
+            vm = koinViewModel { parametersOf(androidContext.packageName) },
+            contract = RequestInstallAppsContract,
+            title = stringResource(R.string.ask_permission_installation),
+            description = stringResource(R.string.ask_permission_installation_description),
+            icon = Icons.Outlined.BugReport
+        )
```

Comment:
Why was this moved here? For context, the bug on android 11 means that apps will be killed if the user grants them installation perms, so it seems better to do it earlier.